odbc_fdw.c

```diff
@@ -1343,6 +1336,8 @@ odbcBeginForeignScan(ForeignScanState *node, int eflags)
 static TupleTableSlot *
 odbcIterateForeignScan(ForeignScanState *node)
 {
+    EState	   *estate = node->ss.ps.state;
```

Comment:
a suggestion after a quick check of the code documentation. At first sight `estate` sounded to me like a Spaniard "false friend". I'd rather prefer long descriptive names vs cryptic short names